Configuration/Geometry/python/dictRun4Geometry.py

```diff
@@ -1384,6 +1528,7 @@
     ("O9","T38","C24","M11","F8","I17") : "D113",
     ("O9","T39","C19","M11","F8","I17") : "D114",
     ("O9","T35","C20","M11","F8","I17") : "D115",
+    ("O9","T35","C25","M12","F9","I17") : "D116",
```

Comment:
Making this detector will make a difficult time to validate M12, as we needs to wait for C25. I would suggest adding another scenario with D110 + M12, so that we can validate M12 first while we are waiting new calorimeter to go for validation.

Reply:
M12 removes the overlaps in M11 and the only sensible validation is to use the g4 overlap tool which is done. Similarly C25 and F9 only remove overlap and that is tested with g4overlap.No other validation makes sense.

Reply:
Did you make it clear what is C25 in README? No, you don't.

Reply:
C25 is derived from C18 where Geometry/EcalCommonData/data/ebalgo.xml is replaced with Geometry/EcalCommonData/data/ebalgo/2021/v1/ebalgo.xml. There is no change in HGCal geometry. The changes in ebalgo.xml is mum and fm are defined in ebalgo/2021/v1/ebalgo.xml. and the type of SpmCutTM and SpmCutTP were not defined in Geometry/EcalCommonData/data/ebalgo.xml  and are defined as "numeric" in the new version. 
In any case we try to see a scenario with very little overlap and going toward 0 overlap. I shall define them in README in a new PR now that the other Run3 PR is combined.